components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/StackLogger.cs

```diff
@@ -35,15 +36,15 @@ public class StackLogger : IStartable, IDisposable {
             Utils.SetTraceMask(0);
             Utils.SetTraceOutput(Utils.TraceOutput.Off);
 
-            // Register callback
-            Utils.Tracing.TraceEventHandler += Tracing_TraceEventHandler;
+            // Set ILogger
+            var microsoftLogger = new SerilogLoggerFactory(Logger)
+                .CreateLogger("OpcUa");
```

Comment:
new logging engine :-)

Reply:
@mregen, I'm a bit confused by your comment. Do I still need to do something for the new logging engine integration?